server/src/main/java/org/apache/iotdb/db/conf/IoTDBDescriptor.java

```diff
@@ -701,19 +701,23 @@ private void loadProps() {
       }
 
       conf.setAuthorizerProvider(
-          properties.getProperty(
-              "authorizer_provider_class",
-              "org.apache.iotdb.db.auth.authorizer.LocalFileAuthorizer"));
+          properties.getProperty("authorizer_provider_class", conf.getAuthorizerProvider()));
       // if using org.apache.iotdb.db.auth.authorizer.OpenIdAuthorizer, openID_url is needed.
-      conf.setOpenIdProviderUrl(properties.getProperty("openID_url", ""));
+      conf.setOpenIdProviderUrl(properties.getProperty("openID_url", conf.getOpenIdProviderUrl()));
```

Comment:
The default value for `openIdProviderUrl` is `null`, which is different with `""`.
It seems that here will throw an exception:
![image](https://user-images.githubusercontent.com/32640567/119110655-f972c680-ba54-11eb-9931-57257af82400.png)

Reply:
Thanks xinyu
do it.